.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,95 @@
+###########################################################################################################
+#
+# This job creates a version bump branch based on master, and opens a PR from it.
+#
+# The branch will have the version bumped in all necessary parts of the code,
+# and it will generate the NEWS.md content from the NEWS snippets.
+#
+# Organization members have the chance to add or modify the commits on the branch, if necessary.
+#
+# The job will also generate a tarball, and reference it in the PR's description.
+#
+# The job cannot be started, if there is an open PR with the version given.
+#
+# Manual steps:
+#  1. Navigate to https://github.com/syslog-ng/syslog-ng/actions.
+#  2. Choose the "Version bump" job.
+#  3. Click "Run workflow".
+#  4. Fill the new release version, and click "Run workflow".
+#  5. When the PR is opened, double check the changes, if necessary, add commits.
+#  6. When the PR is merged, the "Draft release" job will automatically start.
+#
+###########################################################################################################
+
+name: Version bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'VERSION'
+        required: true
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.PAT_FOR_ACTIONS }}
+      RELEASE_VERSION: ${{ github.event.inputs.release_version }}
+      VERSION_BUMP_BRANCH: version/${{ github.event.inputs.release_version }}
+      CURRENT_RUN_URL: https://github.com/${{ github.repository_owner }}/syslog-ng/actions/runs/${{ github.run_id }}
+    steps:
+      - name: Checkout syslog-ng source
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Check existing PR
+        run: |
+          EXSISTS=$(hub pr list --state=open --head=${VERSION_BUMP_BRANCH} | wc -l)
```

Comment:
I really like the safety-checks (check if same version(branch) already opened in a PR, is something merged on master while the version bump PR is open) :+1: 

This is just an additional topic, and I'm interested in your opinion.
I wouldn't add it tho this PR as I see the heavy effort on manual testing the release-automation logic.

The idea is: we check for same version is opened as a PR already, but we can check if additional version bump PRs are open as well (by checking number of open PRs with "version-bump" label).
This is the logic from `.github/workflows/comment-on-version-bump-pr.yml`
```
      - name: Check if version bump PR is open                                                                                                                                                                                               
        run: |                                                                                                                                                                                                                               
          PR_NUMBER=$(hub pr list --state=open --format="%I %l%n" | grep "version-bump" | grep -Po "^[0-9]+") || true                                                                                                                        
                                                                                                                                                                                                                                             
          if [ -n ${PR_NUMBER} ]                                                                                                                                                                                                             
          then                                                                                                                                                                                                                               
            echo "::set-env name=PR_NUMBER::${PR_NUMBER}"                                                                                                                                                                                    
          else                                                                                                                                                                                                                               
            echo "No version bump PR is open. Skipping."                                                                                                                                                                                     
          fi                              
```
Again, this is just an idea for the future. :)

Reply:
Thanks :)

The check here is mainly to make sure we do not overwrite a progress of an ongoing PR, by force pushing to its branch.
We can check if any version bump PR is opened, but I don't believe having multiple open version bump PRs would cause too much trouble, apart the commenting script sending comment to only one of them. I'd say add this later, if we bump into this scenario.

Reply:
Yeah, I agree. Very rare case and merging is done manually, so we would see multiple version bump PRs.
The idea popped up since the "version bump" jobs is manually started, and somebody could accidentally start a new release.
But since the same release cannot be started twice, I think we have everything we actually need. :+1: